.github/workflows/cypress-e2e-parallelization.yml

```
@@ -65,7 +65,39 @@ jobs:
       - name: Cypress e2e testing
         uses: ./.github/actions/cypress-e2e-testing
         with:
+          name: cypress-e2e
           image: docker.centreon.com/centreon-web-centos7
           image_version: ${{ github.head_ref || github.ref_name }}
           module: centreon
           feature_file_path: ${{matrix.feature}}
+
+      - name: Upload e2e tests Results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cypress-e2e-test-reports
+          path: centreon/tests/e2e/cypress/results/reports/*.xml
+
+      - name: Delete all summary content
+        run: rm $GITHUB_STEP_SUMMARY
+        shell: bash
+
+  cypress-e2e-test-report:
+    needs: [cypress-e2e-test-run]
+    if: always()
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: cypress-e2e-test-reports
+          path: cypress-e2e-xunit-reports
+
+      - name: Publish tests report
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        if: ${{ always() }}
+        with:
+          check_name: cypress-e2e-tests-report
```

```suggestion
          check_name: e2e-tests-report
```

the check name for legacy should be legacy-e2e-tests-report